test/system/kubehound_dump.yaml

```diff
@@ -29,11 +29,11 @@ builder:
 # Ingestor configuration (for KHaaS)
 ingestor:
   blob:
-    bucket: "" # (i.e.: s3://<your_bucket>)
+    bucket_name: "" # (i.e.: s3://<your_bucket>)
```

Comment:
Given the fact that we require `s3://`, I would prefer `bucket_url` for the var name